File: sample-codes-for-test/a-fizzbuzz/fizzbuzz.c
#include <stdio.h>

int main() {
        int N;
        scanf("%d", &N);

        for (int i = 0; i < N; ++i) {
                int t;
                scanf("%d", &t);

                if(t % 15 == 0) {
                        printf("FizzBuzz\n");
                } else if(t % 5 == 0) {
                        printf("Buzz\n");
                } else if(t % 3 == 0) {
                        printf("Fizz\n");
                } else {
                        printf("%d\n", t);
                }
        }
}

File: sample-codes-for-test/a-fizzbuzz/fizzbuzz.pl
#!/usr/bin/perl

$N = <>;

for ($i=0; $i<$N; ++$i) {
	$t = int(<>);

	if ($t % 15 == 0) {
		print "FizzBuzz\n";
	} elsif ($t % 5 == 0) {
		print "Buzz\n";
	} elsif ($t % 3 == 0) {
		print "Fizz\n";
	} else {
		print $t . "\n";
	}
}
